package.json

```diff
@@ -10,13 +10,13 @@
   },
   "scripts": {
     "start": "node -r ./tracing.js index.js",
-    "start:local": "OTEL_EXPORTER_OTLP_ENDPOINT=\"http://localhost:4317\" OTEL_RESOURCE_ATTRIBUTES=service.name=node-js-sample-app node -r ./tracing.js index.js"
+    "start:local": "OTEL_EXPORTER_OTLP_ENDPOINT=\"http://stagingapp.signoz.io:4318/v1/traces\" OTEL_RESOURCE_ATTRIBUTES=service.name=node-js-sample-app node -r ./tracing.js index.js"
```

Comment:
can you revert this to `localhost:4317`?

Reply:
localhost:4317 or 4318? the tracing.js file of this app shows 4318 @prashant-shahi

Reply:
okay, reverted back to localhost:4317